.golangci.yml

```diff
@@ -1,3 +1,107 @@
+run:
+  timeout: 5m
+  go: 1.18
 linters:
   disable:
     - errcheck
+    # TODO: enable when fixed
+    - whitespace
+    - staticcheck
+# Enabled by your configuration linters:
```

Comment:
Why do we have this comments here?

Reply:
this are the linters enabled by our configurations. I thought it would be interesting to have them here as a comment to know which ones are being applied. 

the TODO ones i disabled them as there where too many on code and this would need some PRs to clean the errors and enable them again